x/exchange/spec/01_concepts.md

```diff
@@ -109,6 +118,23 @@ E.g. If an order's funds are in a sanctioned account, settlement of that order w
 Or, if a marker has required attributes, but the recipient does not have those attributes, settlement will fail.
 
 
+### Commitment Settlement
+
+A market can move funds committed to it by using the [MarketCommitmentSettle](03_messages.md#marketcommitmentsettle) endpoint.
+
+During commitment settlement:
+
+1. Holds are released on the funds to be transferred.
+2. The funds are transferred
+3. The funds are re-committed in the destination accounts.
+
+The market can collect fees as part of this settlement, but is also charged extra fees for the tx.
+gSee also: [Commitment Fees](#commitment-fees).
```

Comment:
You have a typo here, and this should be `See`.

Reply:
Fixed.